docs/user_guide/source/setting_up/package.rst

```diff
@@ -6,23 +6,37 @@ As Package
 Conda
 *****
 
-Currently ADIOS 2 can be obtained from anaconda cloud:
+The ADIOS 2 Python bindings can be obtained from anaconda cloud:
 
-* x86-64 and MacOS: `williamfgc adios2-openmpi adios2-mpich adios2-nompi <https://anaconda.org/williamfgc>`_  
+* x86-64 and MacOS: `williamfgc adios2-openmpi adios2-mpich adios2-nompi <https://anaconda.org/williamfgc>`_
 * Multiple archs: `conda-forge adios2 <https://anaconda.org/conda-forge/adios2>`_
 
+A recently tested install sequence is
```

Comment:
I wouldn't rely on one-point statistics. Let's double-check this is reproducible on other Mac systems and it's not a configuration issue.